sdk/digitaltwins/test-resources.json

```diff
@@ -1,45 +1,41 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "baseName": {
+    "testApplicationOid": {
       "type": "string",
-      "minLength": 6,
-      "maxLength": 50,
-      "defaultValue": "[resourceGroup().name]",
       "metadata": {
-        "description": "The base resource name."
+        "description": "The client OID to grant access to test resources."
       }
     },
-    "testApplicationOid": {
+    "baseName": {
       "type": "string",
+      "minLength": 6,
+      "maxLength": 50,
       "metadata": {
-        "description": "The client OID to grant access to test resources."
-      }
+        "description": "The base resource name."
+      },
+      "defaultValue": "[resourceGroup().name]"
     },
     "location": {
       "type": "string",
-      "defaultValue": "[resourceGroup().location]",
       "metadata": {
         "description": "The location of the resource. By default, this is the same as the resource group."
-      }
+      },
+      "defaultValue": "[resourceGroup().location]"
     }
   },
+  "functions": [],
   "variables": {
-    "digitalTwinInstanceResourceId": "[resourceId('Microsoft.DigitalTwins/digitalTwinsInstances', parameters('baseName'))]",
-    "namespaceResourceId": "[resourceId('Microsoft.EventHub/namespaces', parameters('baseName'))]",
-    "eventHubResourceId": "[resourceId('Microsoft.EventHub/namespaces/eventhubs', parameters('baseName'), parameters('baseName'))]",
-    "eventHubAuthorizationResourceId": "[resourceId('Microsoft.EventHub/namespaces/eventhubs/authorizationRules', parameters('baseName'), parameters('baseName'), 'owner')]",
-    "rbacOwnerRoleDefinitionId": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/', '8e3af657-a8ff-443c-a75c-2fe8c4bcb635')]",
-    "AdtOwnerRoleDefinitionId": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/', 'bcd981a7-7f74-457b-83e1-cceb9e632ffe')]"
+    "roleDefinitionId": "[format('/subscriptions/{0}/providers/Microsoft.Authorization/roleDefinitions/8e3af657-a8ff-443c-a75c-2fe8c4bcb635', subscription().subscriptionId)]"
   },
   "resources": [
     {
       "type": "Microsoft.Authorization/roleAssignments",
       "apiVersion": "2018-09-01-preview",
       "name": "[guid(resourceGroup().id)]",
       "properties": {
-        "roleDefinitionId": "[variables('rbacOwnerRoleDefinitionId')]",
+        "roleDefinitionId": "[variables('roleDefinitionId')]",
```

Comment:
Seems like the old name was more descriptive.

Reply:
Agreed. I just missed keeping the old name. Will fix it.